source/directives/select-field.ts

```diff
@@ -298,6 +302,7 @@ module formFor {
 
           // Rather than sanitizing `$scope.model.bindable` to undefined, update the empty option's value.
           // This way users are able to choose between undefined, null, and empty string ~ see #141
+          $scope.model.bindable = formForConfiguration_.defaultSelectEmptyOptionValue;
```

Comment:
Hm! Interesting. I *think* it's okay but I haven't been in this code for a while. I assume you've tested it pretty rigorously? If so, I'm inclined to give it a shot. I checked out your Plunker and it didn't seem to have any obvious problems.